### PR TITLE
📝 Document using code climate locally

### PR DESCRIPTION
**Pull request by muchzill4**
### Description

Documenting local usage of Code Climate.

### Test Coverage

- [ ]  This change is covered by existing or additional automated tests.
- [ ]  Manual testing has been performed (and evidence provided) as automated testing was not feasible.
- [x]  Additional tests are not required for this change (e.g. documentation update).

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -95,3 +96,15 @@ relevant place using the [reST include](https://docutils.sourceforge.io/docs/ref
 Type hints should be used in the code wherever possible. Since the 
 documentation shows the function signatures with the type hints 
 there is no need to include additional type information in the docstrings.
+
+
+## Code Climate
+
+Code Climate is integrated with our GitHub flow. Failing the configured rules will yield a pull request not mergable.
+
+If you prefer to view Code Climate report on your machine, prior to sending a pull request you can use the [cli provided by Code Climate](https://docs.codeclimate.com/docs/command-line-interface)
```
Comment (rosiemiller2) [inline DEVELOPMENT.md]:
```suggestion
If you prefer to view the Code Climate report on your machine, prior to sending a pull request, you can use the [cli provided by Code Climate](https://docs.codeclimate.com/docs/command-line-interface).
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -95,3 +96,15 @@ relevant place using the [reST include](https://docutils.sourceforge.io/docs/ref
 Type hints should be used in the code wherever possible. Since the 
 documentation shows the function signatures with the type hints 
 there is no need to include additional type information in the docstrings.
+
+
+## Code Climate
+
+Code Climate is integrated with our GitHub flow. Failing the configured rules will yield a pull request not mergable.
```
Comment (rosiemiller2) [inline DEVELOPMENT.md]: Where are we configuring the rules? Aren't we just using CC out of the box?

Comment (muchzill4) [reply DEVELOPMENT.md]: We are using defaults, but they're kept in the repo in case we want to change something.

Comment (rosiemiller2) [reply DEVELOPMENT.md]: Do you reckon we could point people there so they can see what rules we use? The yml file isn't particularly readable, so maybe not :s

Comment (muchzill4) [reply DEVELOPMENT.md]: I think it's fine as is. If you end up doing something about the file, you'll need to read actual CC docs. Plus I'd rather keep only as much docs as necessary. :)

**Pull request merged**